shell/config/product/apps.js

```diff
@@ -75,9 +75,18 @@ export function init(store) {
     dashIfEmpty: true,
   };
 
+  const clusterRepoBranch = {
+    name:        'branch',
+    labelKey:    'tableHeaders.branch',
+    sort:        'spec.gitBranch',
+    getValue:    (row) => row,
```

Comment:
doesn't this break local sorting / searching?

Reply:
it doesn't

https://github.com/user-attachments/assets/953f2c50-a9f1-406c-85ae-3052236d7552

Reply:
This breaks search (n/a provides no hits).

Would suggest this, which means the formatter might not be neeed

```
  const clusterRepoBranch = {
    name:        'branch',
    labelKey:    'tableHeaders.branch',
    getValue:    (row) => row?.isOciType ? this.t('generic.na') : row?.spec?.gitBranch || undefined,
    dashIfEmpty: true,
  };
```

would just need to resolve access to `t`, which could worst case come from the model (or change to a branchLabel getter)

Reply:
`store` was already accessible so I used `getters` from it. Did you intentionally remove `sort` in your suggested code?